packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -83,7 +84,7 @@ export async function fetchSnapshotWithRedeem(
         snapshotDownloader,
         putInCache,
     ).catch(async (error) => {
-        if (isRedeemSharingLinkError(odspResolvedUrl, error)) {
+        if (enableRedeemFallback && isRedeemSharingLinkError(odspResolvedUrl, error)) {
```

Comment:
This is breaking change, is this fine? We should only make additive changes in release branches.
It would not be breaking if we reverted it, i.e. !disableRedeeemFallback - not providing disableRedeeemFallback  would fallback to existing behavior.

Reply:
The redeeming is not currently working and server people are investigating it. I disabled for all for now so that every host does not need to make the change considering it is not working in most cases.